geopattern/index.d.ts

```diff
@@ -0,0 +1,76 @@
+// Type definitions for GeoPattern 1.2
+// Project: https://github.com/btmills/geopattern
+// Definitions by: Gaelan <https://github.com/Gaelan>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <references types="jquery" />
```

Comment:
Ah. I was running tsc manually rather than with the tsconfig. (AFIACT, neither the Handbook's "Publishing" page nor the contributing guidelines it links mentions `npm run new-package` or including a `jsconfig.json` or `tslint.json`, so I added them later after seeing the readme but was still using the manual `tsc` command from my command history)